docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -894,6 +898,9 @@ You can initialize the shuffle generator from a seed to generate a random-but-id
 
 The shuffle filter returns a list whenever possible. If you use it with a non 'listable' item, the filter does nothing.
 
+.. note::
+
+  While using ``seed`` parameter, the resulting value can be different depending on the version of Python used on the Ansible controller.
```

Comment:
```suggestion
  If you use the ``seed`` parameter, you will get a different result with Python 3 and Python 2. This may break procedures like password generation when you upgrade the version of Python used on your Ansible controller.
```